### PR TITLE
Nothing Productive yet

### PR DESCRIPTION
**Pull request by InnocentBug**
Nothing really productive yet, just a few things that I noticed while going through the code.

### CODE REVIEW

**Comment by InnocentBug** [comment]
Also there are few lines in the tests, that I think should work, but do not work yet.

File: tests/api_resources/test_cript.py [hunk 1]
```diff
@@ -7,251 +7,285 @@
...
-        assert exp1.get("name") == generic_experiment
+    def test_collection_uuid(self, collection_uuid, generic_collection) -> None:
+        col1 = cript.Collection(uuid=collection_uuid)
+        assert col1.get("name") == generic_collection
 
+    def test_create_collection_exisiting_project(self, generic_collection, project_uuid) -> None:
+        col1=cript.Collection(name=generic_collection)
+        proj = cript.Project(uuid=project_uuid, collection=[col1])
+        # TODO I expect this to work
+        # assert proj.collection[0].name == generic_collection
```
Comment (brili) [inline tests/api_resources/test_cript.py:83]: children are hidden in self.__dict__["children"], as I said right now it is upsert only, this functionality needs to be added in order to have that. Plus you would need to find the index of the object.

File: tests/api_resources/test_cript.py [hunk 2]
```diff
@@ -7,251 +7,285 @@
 
 import pytest
 
-from cript import *
+import cript
```
Comment (brili) [inline tests/api_resources/test_cript.py:10]: If you don't want to import all import every used class

File: src/cript/nodes/main.py [hunk 3]
```diff
@@ -26,7 +26,7 @@ def __init__(self, *args, **kwargs):
         self.__dict__["parent"] = None
         schema = copy.deepcopy(cript_schema)
         schema["$ref"] = f"#/$defs/{self.__class__.__name__}Post"
-        resolver = None
+        resolver = kwargs.get("resolver", None)
```
Comment (brili) [inline src/cript/nodes/main.py:29]: this file will get a major overhaul, avoid refactoring at this stage

File: tests/api_resources/test_cript.py (hunk 1 above)
Comment (InnocentBug) [reply tests/api_resources/test_cript.py:83]: okay, we can insert this later and make it work.

I would like to leave the comment in here as a reminder.

File: tests/api_resources/test_cript.py (hunk 2 above)
Comment (InnocentBug) [reply tests/api_resources/test_cript.py:10]: yes, I prefer namespaces clean

File: src/cript/nodes/main.py (hunk 3 above)
Comment (InnocentBug) [reply src/cript/nodes/main.py:29]: sure, but that doesn't hurt right now, does it?

**Comment by brili** [comment]
closing this PR, please submit a clean PR that adds features to the product.

**Pull request closed without merging**